docs/style.md

```diff
@@ -0,0 +1,188 @@
+# InSpec style guide
+
+This is the set of recommended InSpec styles you should use on your daily journey when writing controls.
+
+## Control files
+
+### 1. All controls should be located in the "controls" directory and end in ".rb"
+
+Reason: Most syntax highlighters will render InSpec files correctly across a wide list of tools.
+
+Avoid: `controls/ssh_config`
+Use: `controls/ssh_config.rb`
+
+Avoid: `controls/ssh/config.rb`
+Use: `controls/ssh_config.rb`
+
+### 2. Avoid using "controls" or "control" in the name of your control files
+
+Reason: Using `controls` in the filename again duplicates it and creates unnecessary clutter when reading it. Keep the names short and concise.
+
+Avoid: `controls/ssh_controls.rb`
+Use: `controls/ssh.rb`
+
+
+## Code style
+
+### 3. Use Ruby style conventions
+
+Reason: There are many opinions on how to style code across languages and frameworks. To reduce the differences and make all code more easily readable and understandable we choose to follow some common Ruby style guidelines. The following list is a subset of the current best practices. Since InSpec is focused on writing tests instead of arbitrary programming code we can focus on a subset that is important to test authors.
+
+This is a subset of style conventions taken from [the Ruby style guide](https://github.com/rubocop-hq/ruby-style-guide):
+
+- Use: UTF-8 as source file encoding.
+- Use: 2 spaces per indentation level. No hard tabs
+- Use: Unix-style file endings.
+- Avoid: multiple expressions per line.
+- Use: Spaces around operators, after commas, colons and semicolons.
+```ruby
+sum = 1 + 2
+```
+- Avoid: Spaces after `(`, `[` or before `]`, `)`. Use spaces around `{` and before `}`.
+```ruby
+# bad
+some( arg ).other
+[ 1, 2, 3 ].each{|e| puts e}
+
+# good
+some(arg).other
+[1, 2, 3].each { |e| puts e }
+```
+- Avoid: several empty lines in a row
+
+Please refer to the full guide for the recommended style conventions.
+
+## Controls
+
+### 4. Do not wrap controls in conditional statements
+
+Reason: This will create dynamic profiles whose controls depend on the execution. The problem here is that we cannot render the profile or provide its information before scanning a system. We want to be able to inform users of the contents of their profiles before they run them. It is valid to skip controls that are not necessary for a system, as long as you do it via `only_if` conditions. Ruby's internal conditionals will hide parts of the profile to static analysis and should thus be avoided.
+
+Avoid:
+```ruby
+if package('..').installed?
+  control "package-test1" do
+    ..
+  end
+end
+```
+
+Use:
+```ruby
+control "package-test1" do
+  only_if { package('..').installed? }
+end
+```
+
+Avoid:
+```ruby
+case inspec.platform.name
+when /centos/
+  include_controls 'centos-profile'
+...
+```
```

Comment:
You have newlines in your other examples...not sure how it affects rendering though.